### PR TITLE
Upgrade libraries versions

### PR DESCRIPTION
**Pull request by danpersa**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -12,6 +12,7 @@ version       := "0.0.1"
 mainClass in Compile := Some("org.zalando.spearheads.innkeeper.Innkeeper")
 
 scalaVersion := "2.11.7"
+//scapegoatVersion := "1.1.0"
```
Comment (c00ler) [inline build.sbt]: Why to leave this comment in the code?

File: build.sbt [hunk 2]
```diff
@@ -21,20 +22,30 @@ resolvers += "softprops-maven" at "http://dl.bintray.com/content/softprops/maven
...
+
   "com.google.inject"        % "guice"                                % "4.0",
   "net.codingwell"          %% "scala-guice"                          % "4.0.0",
-  "org.postgresql"           % "postgresql"                           % "9.3-1103-jdbc41",
-  "com.github.tminglei"     %% "slick-pg"                             % "0.9.1",
-  "com.zaxxer"               % "HikariCP"                             % "2.4.1",
+  "org.postgresql"           % "postgresql"                           % "9.4-1206-jdbc42",
+  "com.github.tminglei"     %% "slick-pg"                             % slickPgV,
+  "com.github.tminglei"     %% "slick-pg_date2"                       % slickPgV,
+  "com.zaxxer"               % "HikariCP"                             % "2.4.3",
```
Comment (c00ler) [inline build.sbt]: Shouldn't it be "java6" version?

File: build.sbt (hunk 1 above)
Comment (danpersa) [inline build.sbt]: because I upgraded the scapegoat and then I wanted to remove it...

File: build.sbt (hunk 2 above)
Comment (danpersa) [inline build.sbt]: why?

Comment (danpersa) [inline build.sbt]: From the HikariCP documentation:

```
Java 7 and Java 8 maven artifact:

    <dependency>
        <groupId>com.zaxxer</groupId>
        <artifactId>HikariCP</artifactId>
        <version>2.4.3</version>
    </dependency>

Java 6 maven artifact (maintenance mode):
    <dependency>
        <groupId>com.zaxxer</groupId>
        <artifactId>HikariCP-java6</artifactId>
        <version>2.3.13</version>
    </dependency>
```

Comment (c00ler) [inline build.sbt]: Right. As I remember before they had java8 version and for java6/java7 there was a separate version with "-java6" suffix. Looks like that they changed it.

File: build.sbt (hunk 1 above)
Comment (c00ler) [inline build.sbt]: Then remove it :)

**Comment by c00ler** [comment]
lgtm

**Pull request merged**